pkgs/development/libraries/nss/generic.nix

```diff
@@ -135,6 +147,14 @@ stdenv.mkDerivation rec {
     ++ lib.optionals stdenv.hostPlatform.isILP32 [
       "-DNS_PTR_LE_32=1" # See RNG_RandomUpdate() in drdbg.c
     ]
+    ++ lib.optionals stdenv.hostPlatform.isFreeBSD [
+      "-D_XOPEN_SOURCE=700"
+      "-D__BSD_VISIBLE"
+      # uses compiler intrinsics
+      "-mavx2"
```

Comment:
Does that mean it only runs on avx2 CPUs?

Reply:
No, there are runtime checks for cpu feature bits.

Reply:
We should put that in as part of the comment